lisa/features/disks.py

```diff
@@ -71,6 +72,12 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
     def get_resource_disk_mount_point(self) -> str:
         raise NotImplementedError
 
+    def get_resource_disks(self) -> List[str]:
+        return []
+
+    def get_resource_disk_type(self) -> str:
+        return ""
```

Comment:
It should be RESOURCE_DISK_TYPE_SCSI by default. The empty string is confusing. Actually, an enumeration is better to prevent misuse of a type.

Reply:
Fixed